### DISCUSSION TITLE
how to setup ip pool dns on cloudflare

### DISCUSSION THREAD

**Discussion by traderpedroso** [postalserver/postal#2264] [Help with using Postal] [answered]
I have 2 IP on the IP pool, every IP needs to set i hostname

Example on postal webgui for IP POOLS

1.1.1.1   ip1.example.com
2.2.2.2 ip2.example.com

my PTR RDNS is postal.example.com

My DNS pointing on Cloudflare  

postal.example.com to 1.1.1.1
ip1.example.com 1.1.1.1
 ip2.example.com 2.2.2.2

Your IP address 1.1.1.1 is associated with the postal.example.com
Nevertheless, your message appears to be sent from ip1.example.com.

You may want to change your pointer (PTR type) DNS record and the host name of your server to the same value.

```

Here are the tested values for this check:
IP: 1.1.1.1
HELO: ip1.example.com
rDNS: postal.example.com
```

**Comment by willpower232** [comment, marked as answer]
duplicate of #2261
